tests/RobotFramework/tests/cumulocity/registration/cumulocity_ca.robot

```diff
@@ -4,20 +4,36 @@ Library         String
 Library         Cumulocity
 Library         ThinEdgeIO
 
-Test Setup      Custom Setup
-
 Test Tags       theme:c8y    test:on_demand
 
 
 *** Test Cases ***
 Register Device Using Cumulocity CA
+    [Setup]    Custom Setup
     ${credentials}=    Bulk Register Device With Cumulocity CA    ${DEVICE_SN}
     ${DOMAIN}=    Get Cumulocity Domain
     Execute Command    tedge config set c8y.url "${DOMAIN}"
     Execute Command
     ...    tedge cert download c8y --device-id "${DEVICE_SN}" --one-time-password '${credentials.one_time_password}' --retry-every 5s --max-timeout 30s
     Execute Command    tedge connect c8y
 
+Certificate Renewal Service Using Cumulocity Certificate Authority
+    [Setup]    Setup With Self-Signed Certificate
+    ${cert_before}=    Execute Command    tedge cert show | grep -v Status:
+    Execute Command
+    ...    cmd=sudo env C8Y_USER='${C8Y_CONFIG.username}' C8Y_PASSWORD='${C8Y_CONFIG.password}' tedge cert upload c8y
+    ...    log_output=${False}
+    Execute Command    tedge reconnect c8y
+
+    # Enforce a renewal using the service
+    Execute Command    sudo tedge config set certificate.validity.minimum_duration 365d
```

Comment:
Smart way to force a renewal without hacking the service nor the certificate :-)